dataGenerate.js

```diff
@@ -0,0 +1,30 @@
+import short from "short-uuid";
+import fs from "fs";
+
+const genres = [
+  "Mystery",
+  "Science Fiction",
+  "Fantasy",
+  "Romance",
+  "Thriller",
+  "Historical Fiction",
+  "Horror",
+  "Adventure",
+  "Non-fiction",
+  "Biography",
+  "Self-help",
+];
+
+const bookData = [];
+
+for (let i = 0; i < 100; i++) {
```

Comment:
Write this set of code with more object oriented framework. Use classes.

Reply:
done